policyengine_us/variables/gov/states/ne/tax/income/credits/ctc/refundable/ne_refundable_ctc_child_care_enrolled_eligible_child.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_us.model_api import *
+
+
+class ne_refundable_ctc_child_care_enrolled_eligible_child(Variable):
```

Comment:
```suggestion
class ne_ctc_child_enrolled_in_eligible_care_program(Variable):
```

Reply:
@PavelMakarchuk should i remove `refundable` from all variable names

Reply:
Just the specific eligibility files - lets keep it for the general child eligibility and the file amount